cli/src/deploy/mod.rs

```diff
@@ -112,19 +110,19 @@ async fn get_credentials(token: &str) -> HashMap<String, CredentialsConfig> {
   let path = Path::new(file_name);
   let file = File::open(path);
   if let Err(err) = file {
-    eprintln!("Cannot access credentials at {}. Error: {}", file_name, err);
+    let err_str = format!("Cannot access credentials at {}. Error: {}", file_name, err);
+    eprintln!("{}", err_str);
     eprintln!("{}", CONFIG_SETUP);
-    error!("Cannot access credentials. Error: {}", err);
-    client_error(token, "NO_CREDENTIALS_FILE").await;
+    client_error(token, "NO_CREDENTIALS_FILE", Some(&err_str), None).await;
     std::process::exit(1);
   }
   let reader = BufReader::new(file.unwrap());
   let config = from_reader(reader);
   if let Err(err) = config {
-    eprintln!("Invalid credentials. Error: {}", err);
+    let err_str = format!("Invalid credentials. Error: {}", err);
+    eprintln!("{}", err_str);
```

Comment:
What's the reasoning behind these changes to break out the error string into a separate variable in the code?